common/lib/xmodule/xmodule/modulestore/tests/test_mongo.py

```diff
@@ -1,47 +1,48 @@
 """
 Unit tests for the Mongo modulestore
 """
-# pylint: disable=protected-access
+
 # pylint: disable=no-name-in-module
 # pylint: disable=bad-continuation
```

Comment:
I think both of these comments should also move with the `TestCase` import, they seem designed to suppress some pylint warnings that it triggers (or at least used to trigger).